react-client/src/Components/App.js

```diff
@@ -16,6 +15,7 @@ function App() {
     domType: "",
   });
   const [domSwitch, setdomSwitch] = useState(true);
+  const [displayArray , setdisplayArray] = useState("2");
```

Comment:
Redundant. I can discuss this why. But for now you don't need to fix it.

Reply:
removed from app.js had to transfer to a different component to prevent redundancy regarding the children issues earlier